### PR TITLE
nvm: add page

### PR DESCRIPTION
**Pull request by igorshubovych**

### CODE REVIEW

File: pages/common/nvm.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# nvm
+
+> Node.js version manager.
+> Switch between NodeJS versions: system, node, 0.10, 0.12, 4.2.
+
+- Install a specific version of NodeJS
+
+`nvm install {{node_version}}`
+
+- Use a specific version NodeJS in a current shell
```
Comment (waldyrious) [inline pages/common/nvm.md]: I suppose you mean "in the current shell", right?

File: pages/common/nvm.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`nvm use {{node_version}}`
+
+- Set a default NodeJS version
```
Comment (waldyrious) [inline pages/common/nvm.md]: "a default" should probably be "the default" instead.

File: pages/common/nvm.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`nvm alias default {{node_version}}`
+
+- List all available NodeJS versions, also print default
```
Comment (waldyrious) [inline pages/common/nvm.md]: I'd suggest changing "also print default" to "and print the default one".

**Comment by igorshubovych** [comment]
Thanks for feedback. Updated.

**Comment by waldyrious** [comment]
LGTM

**Pull request merged**